Source/TrackableEntities.EF.5/DbContextExtensions.cs

```diff
@@ -207,8 +208,16 @@ public static void ApplyChanges(this InterceptorPool pool, IEnumerable<ITrackabl
                 {
                     // Mark modified properties
                     SetEntityState(context, item, parent, propertyName, EntityState.Unchanged, interceptors);
-                    foreach (var property in item.ModifiedProperties)
-                        context.Entry(item).Property(property).IsModified = true;
+                    var entry = context.Entry(item);
+                    // Exclude properties attributed as NotMapped
+                    var notMappedProps = item.GetType()
```

Comment:
Oh I missed this comment, but I implemented it without a separate method, I think it should be sufficient. Check my last commit please dd99154e2beb3584d938d78d11f856ce78340344.